main/prelims.tex

```diff
@@ -11,3 +11,25 @@ \section{Some section}
 The \emph{von Neumann model} of a computer, also known as the \emph{Princeton architecture} is an architecture for digital computers, which consists of a processing units, containing an ALU and processing registers; a control unit consisting of an instruction register and a program counter; a memory unit which stores both data and instructions; and input-and-output mechanisms.
 \end{definition}
 
+\section{Acronyms and abbreviations}
+
+\newif\ifabbreviations
```

Comment:
We should assume authors know very little LaTeX, and not suggest they rely on conditional compilation and use of LaTeX booleans...